app/assets/stylesheets/farmbot.css.scss

```diff
@@ -1038,3 +1022,12 @@ table thead tr th {
   padding: .5rem;
 }
 
+.margin-top {
+  margin-top: 9px;
+}
+.margin-left {
+  margin-left: 12px;
+}
+.margin-right {
+  margin-right: 12px;
+}
```

Comment:
@roryaronson What test editor are you using right now? If its Sublime, let me know.